src/Tribe/Cost_Utils.php

```diff
@@ -63,6 +63,10 @@ public function get_event_costs( $event ) {
 
 		$costs = tribe_get_event_meta( $event->ID, '_EventCost', false );
 
+		$costs = array_filter( (array) $costs, static function ( $cost ) {
+			return $cost !== '';
+		} );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 0cfd15071ccbfccfa12274ab3536ec42</sub>